Source/WTF/wtf/PlatformHave.h

```diff
@@ -1762,6 +1762,11 @@
 #define HAVE_APPKIT_SCROLLBAR_COLOR_SPI 1
 #endif
 
+#if !defined(HAVE_UIKIT_SCROLLBAR_COLOR_SPI) \
+    && ((PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 260000))
```

Comment:
The combination of `PLATFORM(IOS_FAMILY)` and `__IPHONE_OS_VERSION_MIN_REQUIRED` is strange.

If you want it on all iOS family, list out the individual platforms like below. If not, then use `(PLATFORM(IOS) || PLATFORM(MACCATALYST)` instead of `PLATFORM(IOS_FAMILY)`.